packages/frontend/src/components/Form/ReportForm/ReportForm.tsx

```diff
@@ -239,14 +252,17 @@ const ReportForm: React.FC<ReportFormProps> = ({ closeModal, onNotifyParentAbout
 
         return false
     }
-    
+
     const validateReportForm = async () => {
         let hasError = false
 
         // Check for last report time to prevent spamming
         const lastReportTime = localStorage.getItem('lastReportTime')
 
-        if ((lastReportTime !== null) && Date.now() - new Date(lastReportTime).getTime() < REPORT_COOLDOWN_MINUTES * 60 * 1000) {
+        if (
```

Comment:
what formatter are you using? we should all use the same one, for git blame and reduced changes in diff

Reply:
I am using prettier with the config that is defined in root, for python I am using black with the default config. I was wondering if you have prettier installed or where the formatting changes are coming from.